### ISSUE TITLE
[bug] Cancelling action of choosing new executable resets current executable

### ISSUE THREAD

**Issue by MathijsG** [opentrack/opentrack#245]
I have some games set up to automate the tracking:

![image](https://cloud.githubusercontent.com/assets/529552/10264216/57819afe-6a05-11e5-9c0a-62a0d908a919.png)

When I click '...'  to search the executable (I expected to be directed to the current executable folder) which isn't.

The currently configured excutable gets zero'd.

![image](https://cloud.githubusercontent.com/assets/529552/10264221/94bf87e6-6a05-11e5-861a-eb58b45846cc.png)

**Comment by sthalik** [comment]
@MathijsG and thanks for the effort of _all_ of the reports you've made.

**Comment by MathijsG** [comment]
And is it also easy and possible to start at the location of the current set executable when you press '...'? Now you always start at a default location.

Thanks for making this software :)

**Comment by sthalik** [comment]
ok fine wait some 😄

**Comment by sthalik** [comment]
I can has emoji? 😈

**Comment by MathijsG** [comment]
No :-P but I do see them in my notification e-mail. Weird Github.

**Comment by sthalik** [comment]
strange, works for me on desktop site. but rdp doesn't so I can bring opentrack here. such sad.

**Comment by sthalik** [comment]
@MathijsG we can't jump to current file's location because it's not stored, only the file's name is stored. It has to be that way sadly.

**Comment by sthalik** [comment]
I got VNC set up. Expect hackage.

**Comment by dbaarda** [comment]
Hmm... just read this after talking to someone about my Cygwin/VNC/NX/X2GO
painful experience trying to get qtcreator to work remotely. I documented
my woes here;

http://minkirri.apana.org.au/wiki/DevJournal

It might help understand things if you encounter problems like I did...

On 6 October 2015 at 12:37, Stanisław Halik <notifications@github.com>
wrote:

> I got VNC set up. Expect hackage.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/opentrack/opentrack/issues/245#issuecomment-145719132>
> .
>



-- 
Donovan Baarda <abo@minkirri.apana.org.au>

**Comment by sthalik** [comment]
Why this convoluted experience? I run Qt Creator with Cygwin cross-compiler kit using stock Cygwin-provided mingw-w64 compilers. There are some pathname discrepancies but nothing important to the build process.

**Comment by dbaarda** [comment]
At the time I wanted to run qtcreator on the linux box to have a pure Linux
devel/build environment. My only Linux box was a headless server, and the
workstation with screen/keyboard was my gaming box running windows. I
wanted to avoid dual boot or linux-in-a-VM-on-windows stuff.

What amazed me was how little of what I tried worked. You would think VNC
client or Cygwin x-server on windows box doing remote-desktop access to a
Linux server would be a no-brainer, but I kept finding new ways that it
wouldn't work.

On 6 October 2015 at 13:26, Stanisław Halik <notifications@github.com>
wrote:

> Why this convoluted experience? I run Qt Creator with Cygwin
> cross-compiler kit using stock Cygwin-provided mingw-w64 compilers. There
> are some pathname discrepancies but nothing important to the build process.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/opentrack/opentrack/issues/245#issuecomment-145725563>
> .
>



-- 
Donovan Baarda <abo@minkirri.apana.org.au>
